<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentEditWizardPage</class>
 <widget class="QWizardPage" name="PaymentEditWizardPage">
  <attribute name="title">
    <string>Payment</string>
  </attribute>
  <layout class="QVBoxLayout">
    <property name="margin">
      <number>11</number>
    </property>
    <property name="spacing">
      <number>6</number>
    </property>
    <item>
      <widget class="QLabel" name="TextLabel8_2_2">
        <property name="text">
          <string>
Please enter the amount you pay for principal and interest or leave the field empty to calculate it.</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <spacer name="Spacer70_2">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>21</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <layout class="QHBoxLayout">
        <property name="margin">
          <number>0</number>
        </property>
        <property name="spacing">
          <number>6</number>
        </property>
        <item>
          <spacer name="Spacer76">
            <property name="sizeHint">
              <size>
                <width>100</width>
                <height>0</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
          </spacer>
        </item>
        <item>
          <layout class="QGridLayout">
            <property name="margin">
              <number>0</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item row="0" column="1">
              <widget class="AmountEdit" name="m_newPaymentEdit"/>
            </item>
            <item row="1" column="0">
              <widget class="QLabel" name="TextLabel2_4">
                <property name="text">
                  <string>Current amount</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="1">
              <widget class="QLabel" name="m_paymentLabel">
                <property name="text">
                  <string>1</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="0">
              <widget class="QLabel" name="TextLabel1_5">
                <property name="text">
                  <string>Principal and interest</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <spacer name="Spacer75">
            <property name="sizeHint">
              <size>
                <width>100</width>
                <height>0</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
          </spacer>
        </item>
      </layout>
    </item>
    <item>
      <spacer name="Spacer71">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>19</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <widget class="QLabel" name="TextLabel4_5">
        <property name="text">
          <string>KMyMoney either calculates the new interest rate or the amount for principal and interest. If you know the amount for principal and interest then enter it here.

If KMyMoney should calculate this value for you, then leave the field blank.</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
   <customwidget>
     <class>AmountEdit</class>
     <extends>QLineEdit</extends>
     <header location="global">amountedit.h</header>
   </customwidget>		 
 </customwidgets>
 <tabstops>
 </tabstops>
</ui>
